flake8_quotes/__init__.py

```diff
@@ -30,13 +30,15 @@ def __init__(self, tree, filename='(none)', builtins=None):
 
     @classmethod
     def add_options(cls, parser):
-        parser.add_option('--quotes', default='\'', action='store',
+        parser.add_option('--quotes', default='\'', action='store', dest='inline_quotes',
+                          help='Quote to expect in all files (default: \') (deprecated)')
+        parser.add_option('--inline-quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
         parser.config_options.append('quotes')
```

Comment:
`config_options` isn't the same as the CLI parser. `flake8` supports input via both CLI flags and configuration files (e.g. `setup.cfg`). `add_option` is for the former, `config_options` is for the latter.